src/App.js

```diff
@@ -8,11 +8,29 @@ import Nav from './components/Nav';
 import './App.css';
 
 function App() {
+  const [locations, setLocations] = useState([]);
+
+  var url = 'https://segdeha.com/api/nearby.php?lat=45&lng=-123.456';
+
+  useEffect(() => {
+    fetch(url)
+      .then(function (response) {
```

Comment:
This works fine, but a more modern looking way to write this would be with ES6-style functions such as the following:

    .then(response => {
      return response.json();
    })

Reply:
There's no functional difference in this case, but it just looks a little funny now to see the `function` keyword.

Reply:
Agreed, I took this block from the docs and didn't remeber to update to ES6 syntax. Thanks for catching these